rj_vision_receiver/src/vision_receiver.cpp

```diff
@@ -314,7 +317,9 @@ void VisionReceiver::UpdateGeometryPacket(
             (fieldSize.field_length() / 1000.0f + (fieldBorder)*2),
             (fieldSize.field_width() / 1000.0f + (fieldBorder)*2)};
 
-        config_.updateFieldDimensions(new_field_dim);
+        config_.updateFieldDimensions(
+            rj_convert::convert_to_ros<Field_Dimensions, Field_Dimensions::Msg>(
+                new_field_dim));
```

Comment:
Looking at it now, this is pretty verbose... would it be possible to infer the second argument?

Reply:
I could probably make an association struct that defaults to `::Msg` pretty easily, let me see. (You'd still have to provide one template argument for the `from_ros` case, though).